free-1.md

```diff
@@ -57,12 +106,23 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I would say 90% of the time would be working on the system to parse calendar
+data and design the interface for the app. 
```

Comment:
Agreed; though I think the interface design contributes to the app's suitability for a project since I don't think the design choices are obvious.